10_rest/runnerdave/README.md

```diff
@@ -0,0 +1,92 @@
+# Puppy server
+
+Storage of Puppies with two different storage methods.
+
+## Pre-requisites
+
+You need to have [Go](https://golang.org/doc/install) installed
+
+## Build and install
+
```

Comment:
nit: one blank line is enough in markdown.
I usually run prettier over all my markdown files for uniform formatting. (similar to gofmt).

Reply:
great tip, I installed prettier and applied it to the README, thanks a lot